experimental/segmenter/src/lstm.rs

```diff
@@ -4,21 +4,98 @@
 
 use icu_segmenter_lstm::lstm::Lstm;
 use std::char::decode_utf16;
+use std::str::Chars;
 
 // TODO:
 // json file is big, So I should use anoher binary format like npy.
 // But provided npy uses tensorflow dtype.
 const THAI_MODEL: &[u8; 373466] = include_bytes!(
     "../../segmenter_lstm/tests/testdata/Thai_codepoints_exclusive_model4_heavy/weights.json"
 );
+const BURMESE_MODEL: &[u8; 475209] =
```

Comment:
Note: We will want this to be served by the DataProvider, which will also allow us to operate on more compact data.  This is tracked by https://github.com/unicode-org/icu4x/issues/905